sdk/communication/Azure.Communication.CallAutomation/src/Models/StartDialogOptions.cs

```diff
@@ -39,6 +52,19 @@ public StartDialogOptions(string dialogId, DialogInputType dialogInputType, stri
             DialogId = dialogId;
         }
 
+        /// <summary>
+        /// Creates a new instance of the DialogOptions.
+        /// </summary>
+        /// <param name="dialogInputType"></param>
+        /// <param name="dialogContext"></param>
+        /// /// <param name="dialogId"></param>
```

Comment:
double summary comment here? (/// ///) 
also it would be a good idea to add some descriptions here